frontend_app_file/src/container/File.jsx

```diff
@@ -823,6 +824,15 @@ export class File extends React.Component {
 
   handleCancelSave = () => this.setState({ showInvalidMentionPopupInComment: false })
 
+  handleClickAutoComplete = () => this.setState({
+    autoCompleteFormNewTagActive: false,
+    autoCompleteClicked: true
+  })
+
+  handleClickAddTagBtn = () => this.setState({ displayNewTagForm: true })
+
+  handleClickCloseAddTagBtn = () => this.setState({ displayNewTagForm: false })
```

Comment:
(@MariePrandstetter @charline-rageade) This works very well, just for learning, another way it could have been done:
```suggestion
  handleToggleAddTagForm = () => this.setState(prev => ({ displayNewTagForm: !prev.displayNewTagForm }))
```

Reply:
There's no doubt about it, your proposal is more elegant !

Reply:
done